Makefile

```
@@ -32,12 +32,22 @@ jadx:
 	@echo "Building Graffiti for Jadx"
 	cp backends/jadx/graffiti.jadx.kts out/graffiti_v$(VERSION).jadx.kts
 
-jeb:
+jeb_packed_%:
+	@echo Packing $*
+	mkdir -p backends/jeb/packed && \
+	( (cat backends/jeb/$*.py | awk '/^#/ {print} !/^#/ {exit}') &&\
+	   echo && echo &&\
+	   python -m pybunch -r backends/jeb -e $* -so ) | cat > backends/jeb/packed/$*.py
+
+JEB_SCRIPTS := graffiti graffitiSyncSymbols graphMe graphMeLine graphMeWithEdgeInfo graphMeXrefs graphMeXrefsLine
```

```suggestion
JEB_SCRIPTS := $(shell grep -rlP '^#\?' backends/jeb | sed 's/.*\///;s/\.[^.]*$$//')
```
Search for all files that starts with `#?`